chrome-extension/manifest.ts

```diff
@@ -40,10 +13,10 @@ const manifest = withSidePanel({
   version: packageJson.version,
   description: '__MSG_extensionDescription__',
   host_permissions: ['<all_urls>'],
-  permissions: ['storage', 'scripting', 'tabs', 'notifications'],
+  permissions: ['storage', 'scripting', 'tabs', 'notifications', 'sidePanel'],
```

Comment:
`// Firefox does not support sidePanel`

How about leaving this comment at the above?